docs/zkapps/tutorials/index.mdx

```diff
@@ -76,4 +78,7 @@ Line numbers are provided for convenience. To prevent copying line numbers and c
 
 The API Reference docs are a detailed resource that is useful after you have familiarized yourself with the basics. See the [SnarkyJS Reference](/zkapps/snarkyjs-reference) docs for an in-depth explanation of all the methods, properties, and interfaces available in SnarkyJS.
 
-See the O(1) Labs blog at https://blog.o1labs.org/. For updates about SnarkyJS, see https://blog.o1labs.org/tagged/snarkyjs.
\ No newline at end of file
+See the O(1) Labs blog at https://blog.o1labs.org/. For updates about SnarkyJS, see https://blog.o1labs.org/tagged/snarkyjs.
+
+If you’re just getting started, watch these step-by-step tutorials that go over how to get started building zkApps.
+<ResponsiveVideo src="https://www.youtube.com/embed/videoseries?list=PLItixFkgfjYFw6GPqu6vk4NmclOJpT9lE" />
```

Comment:
```suggestion
If you're just getting started, watch these step-by-step [zkApp Explainer Videos](https://www.youtube.com/playlist?list=PLItixFkgfjYFw6GPqu6vk4NmclOJpT9lE) that go over how to get started building zkApps. Each tutorial includes a link to the corresponding video. While you're on the Mina Foundation YouTube channel, select **Subscribe** so that new videos will show in your Subscriptions feed.
```

Reply:
I like the idea of having the video playlist on the overview page. However, in this case, using the ResponsiveVideo process is not effective for playlists, since only the first video plays.